src/FSharp.Formatting.ApiDocs/GenerateModel.fs

```diff
@@ -286,6 +295,30 @@ type ApiDocMember (displayName: string, attributes: ApiDocAttribute list, entity
               ParameterType=pty
               ParameterDocs=tdocs.TryFind pnm |} ]
 
+  do
+    let knownExampleIds =
+        comment.Examples
+        |> List.choose (fun x -> x.Id)
+        |> List.countBy id
+    for (id, count) in knownExampleIds do
+        if count > 1 then
+            if warn
+            then
+                printfn "%s(%d,%d): warning: duplicate id for example '%s'" m.FileName m.StartLine m.StartColumn id
+            else
+                printfn "%s(%d,%d): error: duplicate id for example '%s'" m.FileName m.StartLine m.StartColumn id
+    for (id, _count) in knownExampleIds do
+        if id.StartsWith "example-" then
```

Comment:
this pass checks for missing example ids where we potentially inserted a dummy id

Reply:
`then` on same line as `if` please per style guide